src/core/libmaven/isotopeDetection.cpp

```diff
@@ -202,6 +164,58 @@ map<string, PeakGroup> IsotopeDetection::getIsotopes(PeakGroup* parentgroup, vec
     return isotopes;
 }
 
+bool IsotopeDetection::filterIsotope(Isotope x, bool C13Flag, bool N15Flag, bool S34Flag, bool D2Flag, float parentPeakIntensity, float isotopePeakIntensity, mzSample* sample, PeakGroup *parentGroup) 
```

Comment:
@shubhra-agrawal Break all lines with no. of character not more than 80

Reply:
@IpankajI Done in future branches.